data/net/minecraft/world/level/levelgen/presets/WorldPreset.mapping

```diff
@@ -1,3 +1,11 @@
 CLASS net/minecraft/world/level/levelgen/presets/WorldPreset
 	METHOD <init> (Ljava/util/Map;)V
 		ARG 1 dimensions
+	METHOD createWorldGenSettings (JZZ)Lnet/minecraft/world/level/levelgen/WorldGenSettings;
+		ARG 1 seed
+		ARG 3 generateFeatures
```

Comment:
Parameter of constructor which this value is passed into is eventually put into a field called `generateStructures`. If you wish, you can also fix the parameter names for the `WorldGenSettings` constructors to reflect that too.